modules/main/bundle.ts

```diff
@@ -9,6 +9,7 @@ export * from '@deck.gl/google-maps';
 export * from '@deck.gl/mesh-layers';
 export * from '@deck.gl/mapbox';
 export * from '@deck.gl/widgets';
+export {DeckGL as DeckGLFiber} from '@deck.gl/react-fiber';
```

Comment:
Note: this was mostly me messing with the local setup to try and get an import to work in the example package. Can likely remove per feedback.